src/java/org/apache/cassandra/db/DisallowedDirectories.java

```diff
@@ -49,14 +50,21 @@ private DisallowedDirectories()
         MBeanWrapper.instance.registerMBean(this, MBEAN_NAME, MBeanWrapper.OnException.LOG);
     }
 
-    public Set<File> getUnreadableDirectories()
+    @Override
+    public Set<java.io.File> getUnreadableDirectories()
```

Comment:
cc @belliottsmith , had to revert the change done in Paxos as it breaks JMX.  I don't see anything using this methods so they are JMX only, so is there any issue with using java's file type?

Reply:
Spoke in Slack, he was cool with me allowing java file

Reply:
yes, this was a mistake made in CEP-10 when `java.io.File` was replaced - it should have been excluded from the change.